src/serialize.h

```diff
@@ -437,7 +437,7 @@ class CFixedBitSet
             vec[p] = (vBytes[p / 8] & (1 << (p % 8))) != 0;
         if (vBytes.size() * 8 != size) {
             size_t rem = vBytes.size() * 8 - size;
-            uint8_t m = (uint8_t)(0xff << rem);
+            uint8_t m = ~(uint8_t)(0xff >> rem);
```

Comment:
you like your weird operators ;)

Reply:
Not really :D But give me alternatives please :D